apps/app.near/widget/Components/Comment/Compose.jsx

```diff
@@ -0,0 +1,118 @@
+if (!context.accountId) {
+  return "";
+}
+
+const item = props.item;
+const rootItem = props.rootItem;
+
+if (!context.accountId) {
+  return "";
+}
+
+const composeData = () => {
```

Comment:
Can you convert this to a "comment" function in sdk.social that takes an item and content?

What's difference between rootItem and Item?

Reply:
I don't know how this works, I just yanked it from near.social and added custom styles

Reply:
Can you check it out please?

Reply:
`rootItem` was not being used, so I removed it